jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/cpu/OverallCPULoadHandler.java

```diff
@@ -0,0 +1,74 @@
+package io.opentelemetry.contrib.jfr.metrics.cpu;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+
+public class OverallCPULoadHandler implements RecordedEventHandler {
+  public static final String SIMPLE_CLASS_NAME = OverallCPULoadHandler.class.getSimpleName();
+  public static final String EVENT_NAME = "jdk.CPULoad";
+  public static final String JVM_USER = "jvmUser";
+  public static final String JVM_SYSTEM = "jvmSystem";
+  public static final String MACHINE_TOTAL = "machineTotal";
+  public static final String JFR_CPU_LOAD_JVM_USER = "jfr.CPULoad.jvmUser";
```

Comment:
Instead of three different metrics, these should be one metric:  `runtime.jvm.cpu.utilization`, with attributes for `user` or `system`, etc.